Add positions section (#162)
volkamerlab/volkamerlab_org: content/positions/_index.md

---
title: Interested in working with us?
date: 2021-03-03
menu:
    main:
        pre: briefcase

>         name: Interested in working with us?

Reviewer: ```suggestion
        name: Working with us?
```

weight: 120
nav: false
---

> ## Master and Bachelor Student / Student Assistant (HiWis)

Reviewer: ```suggestion
## Master and Bachelor Students / Student Assistants (HiWis)
```

At Volkamer Lab, we develop methods at the interface between structural bioinformatics and cheminformatics, mostly applied in the context of computer-aided drug design with a focus on machine learning methods. 

> In the table bellow you can have a view of the current work ongoing, for more details on each subject, please refer to the research section.

Reviewer: ```suggestion
In the table bellow, you find an overview of the current topics in the group, for more details on each subject, please refer to the research section.
```

> If you are interested in pursuing a bachelor’s or a master’s thesis with us, please gather the following information:

Reviewer: ```suggestion
If you are interested in pursuing a bachelor’s or a master’s thesis with us, or are looking for an Hiwi job, please gather the following information:
```

- Transcript
- CV
- Previous thesis (if applicable)
- A brief description (3-4 sentences) of what you want to work on in our group

> - Field of interest (refer to the table below) and specify the student supervisor in your email

Reviewer: ```suggestion
- Field of interest (refer to the table below) - with concrete topic suggestions, if possible - and specify the most fitting supervisor in your email
```

> Please write an email including the Volkamer Lab member you would like to work with and CC Paula (paula.kramer@cs.uni-saarland.de) and Floriane (floriane.odje@uni-saarland.de). 

Reviewer: ```suggestion
Please write an email including the Volkamer Lab member you would like to work with and CC Paula (paula.kramer[at]cs.uni-saarland.de) and Floriane (floriane.odje[at]uni-saarland.de), or our admin (stefanie.wessinger[at]uni-saarland.de). 
```

> **Important:** Do not use volkamer@cs.uni-saarland.de for Master’s/Bachelor’s HiWi position applications and inquiries.

Reviewer: ```suggestion
**Important:** Do not use volkamer[at]cs.uni-saarland.de for Master’s/Bachelor’s/HiWi position applications and inquiries, we are collecting the information centrally. Also note that we are often overbooked with supervisions, thus, make sure that you approach us in advance, so we have time to consider your application in out planning.
```

| Volkamer Lab Member                   | Field                                                                                             | Site       |
|--------------------------------------|--------------------------------------------------------------------------------------------------|------------|
| Paula Linh Kramer (PhD candidate)    | Deep learning, Fragment based drug design, generative AI                                         | SB         |
| Floriane Odje (PhD candidate)        | Machine learning, Image analysis, toxicity and activity prediction, pattern identification                    | SB         |
| Afnan Sultan (PhD candidate)         | Large language models (LLM), Generative AI for non-toxic properties, property prediction, explainable AI | SB         |
| Yanyuan Zhang (PhD candidate)        | Structure based, binding site comparison, Software Development                                   | SB         |
| Hamza Ibrahim (PhD candidate)        | Virtual screening, physics-based ML                                                              | SB         |

> | Dr. Raquel López-Ríos de Castro | Molecular Dynamics, Machine learning                                                             | B          |

Reviewer: ```suggestion
| Dr. Raquel López-Ríos de Castro (PostDoc) | Molecular Dynamics, Machine learning                                                             | B          |
```

> | Dr. Guillermo Pérez Hernández   | Molecular Dynamics, Software Development                                                         | B          |

Reviewer: ```suggestion
| Dr. Guillermo Pérez Hernández (PostDoc)   | Molecular Dynamics, Software Development                                                         | B          |
```

> | Antoine Lacour (Postdoc)             | Virtual Screening, Software Development                                                          | SB         |

Reviewer: ```suggestion
| Antoine Lacour (PostDoc)             | Virtual Screening, Software Development                                                          | SB         |
```
